FileAPI/support/send-file-formdata-helper.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const kTestChars = "ABC~‾¥≈¤･・•∙·☼★星🌟星★☼·∙•・･¤≈¥‾~XYZ";
+
+// formDataPostFileUploadTest - verifies multipart upload structure and
+// numeric character reference replacement for filenames, field names,
+// and field values using FormData and fetch().
+//
+// Uses /fetch/api/resources/echo-content.py to echo the upload
```

Comment:
Should this mention -escaped instead?

Reply:
The main reason I'm using `echo-content-escaped.py` in `send-file-form-helper.js` is that in those tests, the server is returning the request body, which the script is loading in an iframe; and browsers will treat any request as a download if the resource body has [\0\b\v]. I believe this is done this way because passing a `FormData` as the body to `fetch()` will force it to use UTF-8 encoding, so the only way to test the encoding is by doing a form submission navigation. `send-file-formdata-helper.js` doesn't care about actual forms or about encodings other than UTF-8, so it can use `fetch()`.